docs/Using Fleet/Usage-statistics.md

```diff
@@ -25,6 +25,10 @@ Below is the JSON payload that is sent to Fleet Device Management Inc:
   "vulnDetectionEnabled": true,
   "systemUsersEnabled": true,
   "hostsStatusWebHookEnabled": true,
+  "MDMMacOsEnabled": true,
+  "HostExpiryEnabled": true,
+  "MDMWindowsEnabled": false,
+  "LiveQueryDisabled": false,
```

Comment:
Should these be lowercased like the rest of them?

Reply:
Yes, thanks!